script/bulk_import/base.rb

```diff
@@ -639,12 +643,12 @@ def process_raw(original_raw)
     # }
 
     # [QUOTE=<username>;<postid>]
-    raw.gsub!(/\[QUOTE=([^;\]]+);(\d+)\]/i) do
+    raw.gsub!(/\[QUOTE=([^;\]]+);n(\d+)\]/i) do
```

Comment:
Was that `n` intended?

Reply:
It likely was. For this DB the `n` didn't exist in the quotes. I'll revert the change.